src/utils/languageLocalization/contentText.ts

```diff
@@ -212,6 +208,18 @@ export const contentText: LanguageLocalizationType = {
     en: 'Dark',
     ru: 'Темная',
   },
+  hintText: {
```

Comment:
hintText not best option for naming =)
Let's make it more specific - copyHintText or smth like that
hintText sounds more general, you may have another hints in future and the naming may be confusing

Reply:
Done!